docs/further.md

```diff
@@ -271,7 +280,7 @@ export SNAKEMAKE_PROFILE="$HOME/.config/snakemake"
 
 ==This is ongoing development. Eventually you will be able to annotate different file access patterns.==
 
-## Retries - Or Trying again when a Job failed
+## <a name="retries"></a> Retries - Or Trying again when a Job failed
```

Comment:
Are you sure, this manual html link is necessary? Usually, you should be able to link within a markdown document, directly referring to headings (with whitespace replaced by dashes `-`):
https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown

Reply:
let's see. The html is removed in ed0c0d0b02ba902504b5b0fb97bd6fb4c2e6c32f